cmis_field/fields/cmis_folder.py

```diff
@@ -181,10 +184,10 @@ def clean_up_folder(cmis_object_id, backend_id, dbname):
                     try:
                         _repo.getObject(cmis_object_id).deleteTree()
                     except ObjectNotFoundException:
-                        pass
+                        _logger.info("Cannot clean up folder: ObjectNotFoundException")
 
             # remove created resource in case of rollback
-            test_mode = getattr(threading.currentThread(), "testing", False)
+            test_mode = getattr(threading.current_thread(), "testing", False)
             if not test_mode:
                 record.env.cr.after(
```

Comment:
`env.cr.after()` is not supported anymore.
I think this is the way to go, but I would be glad if someone could confirm my intuition.

```
@record.env.cr.prerollback.add
def clean_up_folder():
    db_registry = registry(record.env.cr.dbname)
    with db_registry.cursor() as cr:
        env = api.Environment(cr, SUPERUSER_ID, {})
        backend = env["cmis.backend"].browse(backend.id)
        _repo = backend.get_cmis_repository()
        # The rollback is delayed by an arbitrary length of time to give
        # the GED time to create the folder. If the folder is not properly
        # created at the time the rollback executes, it cannot be deleted.
        time.sleep(0.5)
        try:
            _repo.getObject(value.getObjectId()).deleteTree()
        except ObjectNotFoundException:
            _logger.info("Cannot clean up folder: ObjectNotFoundException")
```